src/v/cluster/types.h

```diff
@@ -1226,7 +1228,14 @@ replication_factor parsing_replication_factor(const ss::sstring& value);
 // This class contains updates for topic properties which are replicates not by
 // topic_frontend
 struct incremental_topic_custom_updates
+<<<<<<< HEAD
```

Comment:
Merge conflict. Did you mean to drop this commit? I find it weird that we set compat version to 0 for `begin_tx_reply` (instead of 1).

Reply:
Yeah, I'll rebase when the fixes land. The begin_tx_reply change is https://github.com/redpanda-data/redpanda/pull/7316 where I guess I was just assuming the struct was _meant_ to be backward compat, but didn't look at the encode/decode to check, I'll paste a link into transactions slack